config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
+      resources :users, only: [:index]
```

Comment:
probably do not need this anymore since the other resources for users are there now.